tests/ssl-opt.sh

```diff
@@ -551,6 +551,12 @@ run_test() {
     CLI_EXPECT="$3"
     shift 3
 
+    # Check if test uses files
```

Comment:
Commit 2e53f38f4 has a typo in the title - "is" -> "if"

Reply:
@AndrzejKurek I prefer not to rewrite history to keep the CI intact. Are you OK with that?

Reply:
It's not a problem for now, but I'd prefer it fixed before merging.